### DISCUSSION TITLE
Window application size get changes when open the CefSharp implemented form open

### DISCUSSION THREAD

**Discussion by abhijeetkoratkar** [cefsharp/CefSharp#4373] [General]
I am having very strange issue with the CefSharp chromium browser implementation under c# window application. We have maximized MDI form and when we open another form as a popup window form which contains the chromium Browser the text size of text changes to smaller and screen size of the window application changes to smaller. Please let me know if anyone has already faced this issue and solution for this?

**Comment by amaitland** [comment]
Please provide more information, a screenshot of the resulting behaviour would likely be helpful.

What `DPI` are you currently using?

See https://github.com/cefsharp/CefSharp/wiki/Questions-&-Support#when-asking-a-question-please-include-the-following

**Reply by abhijeetkoratkar** [reply]
amaitland,

Thank you, your DPI question helped me to check it properly. I was using the scale 125% and by your suggestion tried it to 100% and it worked properly, and I put the Display Resolution to 1366*768 to see the text properly.

Once again thank you for your help.

-Abhijeet

**Reply by abhijeetkoratkar** [reply]
Amaitland,

also let me know if there is any other way to fix this thing without changing the scale from 125% to 100%.
Thanks in Advance.

-Abhijeet

**Reply by abhijeetkoratkar** [reply]
> Please provide more information, a screenshot of the resulting behaviour would likely be helpful.
> 
> What `DPI` are you currently using?
> 
> See https://github.com/cefsharp/CefSharp/wiki/Questions-&-Support#when-asking-a-question-please-include-the-following

Alex,
 I was using the scale 125% and by your suggestion tried it to 100% and it worked properly, and I put the Display Resolution to 1366*768 to see the text properly.
Let me know if there is any other way to fix this thing without changing the scale from 125% to 100%.
Thanks in Advance.

I have attached screenshots.
![MainScreen _1](https://user-images.githubusercontent.com/123197833/214356723-d7f23bbe-fb0d-4132-88e9-a23ee1c70e3b.jpg)
![MainScreenAfterMaximization_3](https://user-images.githubusercontent.com/123197833/214356732-a7b841ee-694d-4619-b4c4-c07f43b4cae8.jpg)
![MainScreenChange_2](https://user-images.githubusercontent.com/123197833/214356735-a111af08-75a6-4f60-aef1-df67da39a3cb.jpg)
![DPI_settings_4](https://user-images.githubusercontent.com/123197833/214356739-8ff61b1d-8d8e-4bb8-9f2f-d3593cf8f4d7.jpg)

**Reply by amaitland** [reply]
See https://github.com/cefsharp/CefSharp/wiki/General-Usage#high-dpi-displayssupport

Either make your application `DPI Aware` or use one of the options in the `Additional Info` section at the bottom to opt-out.

**Reply by abhijeetkoratkar** [reply]
Alex,
I made changes which you suggested, and test application is working properly. But when I make same changes into main application and it still not working and changing the application size and font to smaller.

![image](https://user-images.githubusercontent.com/123197833/217900206-0d0e7a7d-67c3-49aa-9170-3acf12ff4770.png)

![image](https://user-images.githubusercontent.com/123197833/217900320-b70c7f2a-6e4b-47f1-8641-4e539d92bea9.png)

![image](https://user-images.githubusercontent.com/123197833/217900686-ef6a71f0-3ec6-48c3-b97b-5be901fcedff.png)

![image](https://user-images.githubusercontent.com/123197833/217900759-89d663f6-58fa-4164-90d3-501c87ef7e5b.png)

As per your suggestion I have made changes under app.config and under Manifest file and also tried additonal info also but In my main application it is not working.

Thanks in advance!!

-Abhijeet

**Reply by amaitland** [reply]
You can opt-out of requiring your application to be `DPI Aware` using one of the options I've already linked.

**Comment by jquerijero** [comment]
Should a child form with embedded CefSharp browser affect/reset the parent application DPI unawareness/awareness in the first place? Remember that some Windows (particularly servers) versions offer a bitmap scaling to handle non-DPI aware application. What seems to be happening with the original poster's application is that CefSharp is overriding the bitmap scaling.

**Reply by amaitland** [reply]
> Should a child form with embedded CefSharp browser affect/reset the parent application DPI unawareness/awareness in the first place?

It's the default `Chromium` behaviour and outside of my direct control. You can override the behaviour, see https://github.com/cefsharp/CefSharp/issues/4410 for more details.
